packages/pixeloven-webpack-compiler/jestrc.json

```diff
@@ -0,0 +1,41 @@
+{
+  "bail": true,
+  "verbose": true,
+  "preset": "ts-jest",
+  "transform": {
+    "^.+\\.(ts|tsx)$": "ts-jest"
+  },
+  "transformIgnorePatterns": [
+    "[/\\\\]node_modules[/\\\\].+\\.(js|jsx|mjs|ts|tsx)$"
+  ],
+  "testMatch": [
+    "<rootDir>/src/**/test/**/*.(j|t)s?(x)",
+    "<rootDir>/src/**/?(*.)(spec|test).(j|t)s?(x)"
+  ],
+  "testEnvironment": "node",
+  "testURL": "http://localhost",
+  "moduleFileExtensions": [
+    "js",
+    "json",
+    "jsx",
+    "ts",
+    "tsx"
+  ],
+  "coveragePathIgnorePatterns": [
+    "/node_modules/",
+    "/test/"
+  ],
+  "coverageThreshold": {
```

Comment:
Why `0`s here?

Reply:
I don't know how to test this part yet. So for now this is going to be 0 coverage